pipelines/upgrade_pipeline.yml

```diff
@@ -6,6 +6,7 @@
 - import_playbook: upgrade/05-server_to_intermediate.yaml
 - import_playbook: upgrade/06-proxy_to_intermediate.yaml
 - import_playbook: upgrade/07-server_to_final.yaml
-- import_playbook: upgrade/08-proxy_to_final.yaml
-- import_playbook: upgrade/09-tests.yaml
-- import_playbook: upgrade/10-smoker.yaml
+- import_playbook: upgrade/08-tests.yaml
```

Comment:
```suggestion
 - import_playbook: upgrade/08-tests.yaml
+  when:
+    - forklift_upgrade_version_final == 'nightly' or forklift_upgrade_version_final is version('4.1', '>=')
```

Reply:
this wouldn't have worked if run by jenkins (which doesn't use this file)

I pushed a version that should work everywhere